Library/Formula/git-plus.rb

```diff
@@ -0,0 +1,25 @@
+class GitPlus < Formula
+  homepage "https://github.com/tkrajina/git-plus"
+  head "https://github.com/tkrajina/git-plus.git"
+  url "https://github.com/tkrajina/git-plus/archive/v0.1.0.tar.gz"
+  sha256 "d7076ed3fd8e6e4bc2b93b9880adf58bea59b0f6fe95e05053f00247ae8cb609"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    site_packages = libexec/"lib/python2.7/site-packages"
+    libexec_bin = libexec/"bin"
+
+    ENV.prepend_create_path("PYTHONPATH", site_packages)
+
+    site_packages.install("gitutils.py")
+    libexec_bin.install(Dir["git-*"])
+
+    bin.install(Dir[libexec_bin/"*"])
+    bin.env_script_all_files(libexec_bin, :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    shell_output("git multi")
```

Comment:
@mikemcquaid Whoops. Yup, you're right of course, completely forgot about Git's rather nifty subcommand handling; My apologies.

```ruby
  test do
    system "git", "multi"
  end
```

Works though. Tested that earlier, before I wandered off and came up with something more complex for fun :smile_cat:.